recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
   run:
```

Comment:
I wonder if part of the issue is that we need to add rowan and cmeutils? See the comment in #4